core/src/test/java/org/apache/iceberg/catalog/CatalogTests.java

```diff
@@ -168,6 +168,14 @@ protected boolean supportsNamesWithDot() {
     return true;
   }
 
+  protected boolean supportsCreatingEmptyNamespace() {
```

Comment:
currently, the only reason that two flags exist is because Nessie's behavior is different from JDBC + InMemory.